exercises/E-conditionals/exercise.js

```diff
@@ -8,7 +8,11 @@
 
 var name = "Daniel";
 var danielsRole = "mentor";
-
+if ((danielsRole = "mentor")) {
+  console.log("Hi, I'm Daniel, I'm a mentor.");
+} else {
+  console.log("Hi, I'm Daniel, I'm a student.");
+}
 /* 
```

Comment:
An extra challenge for yourself would be to change the code so that your console message returns the correct regardless of what role I assign to Daniel.

Right now, if I change `danielsRole = "teacher"` then it will print out `Hi, I'm Daniel, I'm a student`. What would you change?

Reply:
I think the most efficient way to write the code is without an if block:
```
var name = "Daniel";
var danielsRole = "mentor";
console.log("Hi, I'm  " + name + ", I'm " + danielsRole);
```
But since the exercise requires an if block, I change it to this:  
```
var name = "Daniel";
var danielsRole = "mentor";
if ((danielsRole === "mentor")) {
  console.log("Hi, I'm Daniel, I'm a mentor.");
} else if (danielsRole === "student") {
  console.log("Hi, I'm Daniel, I'm a student.");
} else {
  console.log("Hi, I'm  " + name + ", I'm " + danielsRole)
}
```